src/PrestaShopBundle/Form/Admin/Type/ButtonCollectionType.php

```diff
@@ -84,8 +86,11 @@ public function configureOptions(OptionsResolver $resolver)
                 'label' => false,
                 'buttons' => [],
                 'justify_content' => 'space-between',
+                'inline_buttons_limit' => null,
+                'force_labels' => false,
```

Comment:
```suggestion
                'use_inline_labels' => true,
```
Suggestion of different naming approach, in your use case for combination actions you would then have to set `'use_inline_labels' => false,`